backend/utils/influencerAnalytics.js

```diff
@@ -27,32 +27,38 @@ const trackingData = () => {
 //Instagram
 const InstagramData = async (instagramId) => {
   const options = {
-    method: "GET",
-    url: process.env.INSTA_ENDPOINT,
+    method: 'GET',
+    url: config.INSTA_ENDPOINT,
     params: {
-      username_or_id: instagramId,
-      include_about: "true",
-      nocors: "true",
+      url: `https://www.instagram.com/${instagramId}/`
     },
     headers: {
       "x-rapidapi-key": config.X_RAPIDAPI_KEY,
       "x-rapidapi-host": config.X_RAPIDAPI_HOST_INSTA,
     },
   };
-
+  
   try {
     const response = await axios.request(options);
-    const _data = response.data.data;
+    const data = response.data.data
```

Comment:
we should use optional chaining here to avoid nullPointerExceptions `response?.data?.data`